src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const allLi = document.querySelector('.tree').querySelectorAll('li');
```

Comment:
You don't need to call `querySelectorAll` on the result of `querySelector`. You can combine them into a single call like this: `document.querySelectorAll('.tree li')`. This is more efficient and easier to read.